src/Authentication/Authenticators/AccessTokens.php

```diff
@@ -65,6 +65,15 @@ public function attempt(array $credentials): Result
 
         $user = $result->extraInfo();
 
+        if ($user->banned) {
```

Comment:
Using the property directly makes it impossible to define interface in the future.
I think it is better to have `User::isBanned()` like `isActivated()`:
https://github.com/codeigniter4/shield/blob/develop/src/Traits/Activatable.php#L13